src/ensembl/src/content/app/species/state/general/speciesGeneralHelper.ts

```diff
@@ -651,38 +651,51 @@ export const getStatsForSection = (props: {
       })
     : undefined;
 
+  const processedGroups = groups.map((group) => {
+    const groupStats = groupsStatsMap[group];
+    const expandedStatsKeys: Stats[] = [];
+    const processedStats = groupStats
+      .map((subGroupStats) => {
+        const processedSubGroupStats: IndividualStat[] = [];
+
+        subGroupStats.forEach((stat) => {
+          if (filteredData[stat]) {
+            expandedStatsKeys.push(stat);
+            const individualStat = buildIndividualStat({
+              primaryKey: stat,
+              primaryValue: filteredData[stat],
+              section
+            });
+            individualStat && processedSubGroupStats.push(individualStat);
+          }
+        });
+
+        return processedSubGroupStats.length
+          ? processedSubGroupStats
+          : undefined;
+      })
+      .filter(Boolean);
+
+    const summaryExpandedStatsDifference = expandedStatsKeys?.filter(
+      (key) => !summaryStatsKeys?.includes(key)
+    );
+
+    if (!summaryExpandedStatsDifference.length || !processedStats.length) {
+      return {
+        title: groupTitles[group]
+      };
```

Comment:
Could you remind me of the purpose of these objects, with title, but no stats?

Reply:
It will still display the section with the title and the summary stats are handled separately. There will be no expandedStats to display which will hide the expanded section.